README.md

```diff
@@ -90,13 +90,65 @@
 ## Getting Started
 
 
-### Prerequisites
-
-
-
-### Installation
-
-
+### Setup
+#### First you need is load Alpine.js and Tailwindcss
+
+```bash
```

Comment:
Dal si bash, no pises HTML kod. Ukazuje to divne zafarbenie, skontroluj kde vsade si dal bash a daj tam html.